### PR TITLE
Migrate device account

### PR DESCRIPTION
**Pull request by acosferreira**
# Description

Setting a default account to the existent devices, cClowdapp Should be rollback 

Fixes # (issue)

## Type of change

What is it?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update
- [ ] Refactor

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [X] I run `go fmt ./...` to check that my code is properly formatted
- [ X] I run `go vet ./...` to check that my code is free of common Go style mistakes

### CODE REVIEW

File: cmd/db/set_account_on_device.go [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+		"DefaultOSTreeRef":         cfg.DefaultOSTreeRef,
+		"InventoryURL":             cfg.InventoryConfig.URL,
+		"PlaybookDispatcherConfig": cfg.PlaybookDispatcherConfig.URL,
+		"TemplatesPath":            cfg.TemplatesPath,
+		"DatabaseType":             cfg.Database.Type,
+		"DatabaseName":             cfg.Database.Name,
+	}).Info("Configuration Values:")
+	db.InitDB()
+
+	log.Info("Setting default account to existent devices...")
```
Comment (ldjebran) [inline cmd/db/set_account_on_device.go:36]: I was thinking that we are going to query inventory for the devices using their uuid and get the account info for each device and save it locally

File: Dockerfile [hunk 2]
```diff
@@ -23,6 +23,7 @@ RUN go build -tags=fdo -o /go/bin/edge-api
 # Build the migration binary.
 RUN go build -o /go/bin/edge-api-migrate cmd/migrate/main.go
 RUN go build -o /go/bin/edge-api-wipe cmd/db/wipe.go
+RUN go build -o /go/bin/edge-api-wipe cmd/db/set_account_on_device.go
```
Comment (ldjebran) [inline Dockerfile:26]: may be 
```go
RUN go build -o /go/bin/edge-api-migrate-device cmd/db/set_account_on_device.go
```

File: cmd/db/set_account_on_device.go (hunk 1 above)
Comment (acosferreira) [reply cmd/db/set_account_on_device.go]: I discussed in a thread, as stage data, will be enough just set a default value to the old devices ;)

Comment (acosferreira) [reply cmd/db/set_account_on_device.go]: do you have any concern on this action?

Comment (ldjebran) [reply cmd/db/set_account_on_device.go]: I would prefer if we add "WHERE account IS NULL"

**Review by ldjebran — APPROVED** [review]

**Pull request merged**